package.json

```diff
@@ -70,11 +70,15 @@
     "lavamoat:build:auto": "yarn lavamoat:build --writeAutoPolicy",
     "lavamoat:debug:build": "yarn lavamoat:build --writeAutoPolicyDebug --policydebug lavamoat/build-system/policy-debug.json",
     "lavamoat:background:auto": "./development/generate-lavamoat-policies.sh",
-    "lavamoat:auto": "yarn lavamoat:build:auto && yarn lavamoat:background:auto"
+    "lavamoat:auto": "yarn lavamoat:build:auto && yarn lavamoat:background:auto",
+    "ts-migration:enumerate": "ts-node development/ts-migration-dashboard/scripts/writeListOfFilesToConvert.ts",
+    "ts-migration:dashboard:watch": "ts-node development/ts-migration-dashboard/scripts/build.ts --watch",
+    "ts-migration:dashboard:build": "ts-node development/ts-migration-dashboard/scripts/build.ts",
+    "ts-migration:dashboard:deploy": "gh-pages --dist development/ts-migration-dashboard/build --remote ts-migration-dashboard"
   },
   "resolutions": {
     "**/regenerator-runtime": "^0.13.7",
-    "**/caniuse-lite": "1.0.30001265",
+    "**/caniuse-lite": "^1.0.30001312",
```

Comment:
I was getting warnings about a mismatch between the expected version of `caniuse-lite` and the one that was installed so I bumped this version.